src/notes/StructuredFieldPlugin.jsx

```diff
@@ -51,13 +51,43 @@ function StructuredFieldPlugin(opts) {
 
     function onKeyDown(e, key, state, editor) {
         const {selection} = state;
-
         // We want to consider where the cursor is focused if an expanded selection
         const useFocusKey = selection.isExpanded && selection.isBackward;
         const selectionKey = useFocusKey ? selection.focusKey : selection.anchorKey;
         const parentNode = state.document.getParent(selectionKey);
         const shortcut = parentNode.data.get('shortcut');
 
+        // If the previous node is not an inserter,
+        // delete the full node when hitting backspace right before it, as it is not editable.
+        const previousNode = state.document.getPreviousSibling(state.selection.anchorKey);
+        
+        if (e.key === 'Backspace' && previousNode) {
+            if (previousNode.type === 'structured_field'
+                && !(previousNode.data.get('shortcut') instanceof InsertValue)
+                && state.selection.anchorOffset === 0 && state.selection.isCollapsed) {
+                let transform = state.transform();
+                transform = transform.removeNodeByKey(previousNode.key);
+                let newState = transform.apply();
+                return newState;
+            }
+        } 
+        else if (e.keyCode === 37 && previousNode){
```

Comment:
Unnecessary single space (lines 73 and 81). Should probably just move the `else if (...)` up a line to be on the same line as the bracket to stay consistent with our JS style

Reply:
Also need a space on line 74 before the bracket